yanksel/init.lua

```diff
@@ -5,18 +5,22 @@
 local error = error
 local luakit = luakit
 local lousy = require("lousy")
-local key, buf, but = lousy.bind.key, lousy.bind.buf
-local add_binds, add_cmds = add_binds, add_cmds
+
+local modes = require "modes"
+
+local add_binds = modes.add_binds
+local add_cmds = modes.add_cmds
 
 module("plugins.yanksel")
 
-add_binds("normal", {
-    buf("^ys$",
-	function (w)
-	    local text = luakit.selection.primary
-	    if not text then w:error("Empty selection.") return end
-	    luakit.selection.clipboard = text
-	    w:notify("Yanked selection: " .. text)
-	    luakit.selection.primary = ""
-	end),
-})
+function yank_select(w)
```

Comment:
I'd say make this an action table, which is just a table with `func` (function) and `desc` (description) keys. See `lib/binds.lua` for examples (scrolling actions).